src/bp/ui-studio/src/web/components/Layout/Sidebar.jsx

```diff
@@ -79,10 +69,10 @@ class Sidebar extends React.Component {
       module.menuIcon === 'custom' ? (
         <img className={classnames(style.customIcon, 'bp-custom-icon')} src={iconPath} />
       ) : (
-        <i className="icon material-icons" style={{ marginRight: '5px' }}>
-          {module.menuIcon}
-        </i>
-      )
+          <i className="icon material-icons" style={{ marginRight: '5px' }}>
```

Comment:
you might not have the same auto-format as we do, not sure though

Reply:
right, let's check it out.